### PR TITLE
Gftools packager: several issues

### LINKED ISSUE #233: Slim down upstream.yaml files?
@graphicore we now have 2 upstream.yaml files in the central GF repo,

* https://github.com/google/fonts/blob/master/ofl/literata/upstream.yaml
* https://github.com/google/fonts/blob/master/ofl/chilanka/upstream.yaml

I'd like to propose making these more terse, and eliminate duplication with METADATA.pb files.

```yaml
# Please edit this upstream configuration for the family accordingly.
# This is a yaml formatted file.
# An "#" (number sign) denotes a comment.
# For more help see the docs at:
# https://github.com/googlefonts/gf-docs/tree/master/METADATA
```

I'm not sure we need all these comments in every file; adding it to `/README.md` or `/TRIVIA.md` would be fine, linking to 1 reference file with it all. 

```yaml
name: Literata
repository_url: https://github.com/googlefonts/literata
category: SERIF
designer: TypeTogether
```

Are these all in METADATA.pb?

```
# Choose one of: {', '.join(CATEGORIES)}
```

Bug in the template generator :)

```
files:
  # These are some examples as comments, please modify, add, delete as necessary:
  OFL.txt: OFL.txt
  DESCRIPTION.en_us.html: DESCRIPTION.en_us.html
```

Since these have a 1:1 mapping, I think it would be good to make the key implicit and therefore dropped.

### PR DESCRIPTION
**Pull request by graphicore**
See #238: trying to fix wrongly selected remote for PR

Fixes an issue that happened to me when testing this, unlikely in normal usage though

I don't know if `_pygit2.GitError: unsupported URL protocol` is fixed with this!

UPDATE:

also fix #233 remove redundant info from upstream.yaml, remove comments from upstream.yaml

### CODE REVIEW

**Review by m4rc1e — CHANGES_REQUESTED** [review]
Thank you for adding the OS X fix. We need to make another one for when we fetch the remote. I'll comment on the exact line now.

File: Lib/gftools/packager.py [hunk 1]
```diff
@@ -1772,7 +1841,7 @@ def _git_fetch_master(repo: pygit2.Repository, remote_name: str) -> None:
   # using just 'master' instead of 'refs/heads/master' works as well
   stats = remote.fetch(['refs/heads/master'], callbacks=PYGit2RemoteCallbacks())
```
Comment (m4rc1e) [inline Lib/gftools/packager.py:1842]: This doesn't work on OS X. We get the same ssh issue which your last commit fixed. I just commented out this line and replace it with

```
subprocess.run(['git', 'fetch', 'upstream'],
                             cwd="/Users/marcfoley/Type/fonts",
                             check=True,
                             stdout=subprocess.PIPE) 
```

and it worked, https://github.com/google/fonts/pull/2664.

Comment (graphicore) [reply Lib/gftools/packager.py]: @m4rc1e it's done in cf7bcd4. Maybe you can give it another test run.

Comment (m4rc1e) [reply Lib/gftools/packager.py]: Sure thing.

**Review by m4rc1e — APPROVED** [review]
LGTM. lmk if you want me to merge and I'll tag a new release for pypi.

**Pull request merged**